ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.h

```diff
@@ -149,7 +149,7 @@ without the extension.
 */
 - (UIView *)miniAppViewWithName:(NSString *)name
                      properties:(NSDictionary *_Nullable)properties
-                        overlay:(BOOL)overlay
+                        overlay:(BOOL)overlay;
 /**
```

Comment:
I think there should also be one blank line between declaration and the doc header of the next item?